### DISCUSSION TITLE
Public Generated Process

### DISCUSSION THREAD

**Discussion by PrinceArmah01** [Chanmonich-Phe/Tafesa22S202#2] [General]
I just cloned the repository from GitHub onto my local machine and this is the public key I generated below:

ssh-rsa AAAAB3NzaC1yc2EAAAABJQAAAQEA1w1xPuO0sbdsOOvIKF0NG9wZmDpyjR6CdKgGX7uPakexq3R6cq+st/yqo04UmBK6SBaXUy0ErlGjzhcv9MvUNqljisJl1luW5m5irF1CoQTg6npeEXwTuLWnh3KwrU9Qnm7FvkDYWoJ3xmzG/biAmRzLq1H0Di1eG+ofnu1kGSrdFk8tV68LfjzHcjQDwtcBownuPE/QMQ5eu7E6QKXClVtPl4reqce7bGtBKyE0viJbMCyq0X6zJoQK9buO6hEKA07tPo4NYRCzfzBKQdQDnehgdX5LQH7zTu0szcdZKFw0VtmzbIC2ouQE2DJZTUSwipBS62eDYU3KnwBt7KHqCQ== rsa-key-20220902

**Comment by Chanmonich-Phe** [comment]
Okay thanks, Prince! I'll start adding your generated key to the key ring of the GitHub repository.
